### ISSUE TITLE
Mac OS Copy and Option Key do not work

### ISSUE THREAD

**Issue by arcadian09** [tmux/tmux#1550]
- your platform;
Mac OS Mojave

- a brief description of the problem with steps to reproduce;
Running Tmux on terminal.app, tried all possible combination when `set -g mouse on` the Option key (meta) gets disabled it would seem as nothing occurs when I press it and I tried to highlight to copy. The behaviour is not observed on iterm2.

- a minimal tmux config, if you can't reproduce without a config;
`if-shell '[[ -z ${SSH_CLIENT} ]]' \
    "set -g prefix C-a; unbind C-b; bind C-a send-prefix; set -g status-position top"

# No bells at all
set -g bell-action none

#setting the delay between prefix and command
set -s escape-time 1
# Set the base index for windows to 1 instead of 0
set -g base-index 1
# Set the base index for panes to 1 instead of 0
setw -g pane-base-index 1

# enable vi keys.
#set-window-option -g mode-keys vi
setw -g mode-keys vi

# shortcut for synchronize-panes toggle
##bind C-s set-window-option synchronize-panes

# Set Scroll History
set-option -g history-limit 10000

bind c new-window      -c "#{pane_current_path}"
bind | split-window -h  -c "#{pane_current_path}"
bind - split-window -v  -c "#{pane_current_path}"

set -g mouse on
# Zoom pane
bind + resize-pane -Z

# Kill pane/window/session shortcuts
bind x kill-pane
bind X kill-window

# Detach from session
bind d detach

#-------------------------------------------#
#----------------- Visual ------------------#
# set the status line's colors
set -g status-style fg=white,bg=blue,dim
# Status line right side - host 13:37 | 31-Oct
set -g status-right "#h | %R | %d-%b"

# set the color of the inactive window list
setw -g window-status-style fg=white,bg=blue,dim
# set colors for the active window
setw -g window-status-current-style fg=blue,bg=white,dim

# set the colors for pane borders
setw -g pane-border-style fg=blue,bg=colour239
setw -g pane-active-border-style fg=blue
# Set active an normal, other shaded out
setw -g window-style bg=colour239
setw -g window-active-style bg=colour237
# Command / message line
set -g message-style fg=black,bold,bg=magenta

# enable activity alerts
setw -g monitor-activity on
set -g visual-activity on`

- your terminal, and $TERM inside and outside of tmux;
terminal.app
screen-256color
- logs from tmux (see below);

- at most one or two screenshots, if helpful.

This should include at least the output of:

    $ uname -sp && tmux -V && echo $TERM
Darwin i386
tmux 2.8
screen-256color

**Comment by olegTarassov** [comment]
The behaviour is not observed when running it outside tmux. Inside terminal.app I can select the text with Option key and dragging the mouse for a copy+paste, I can even move the cursor to a different place of the line when holding Option and pressing the left mouse button.

The moment I go into tmux, I lose all capabilities of the Option key it seems. Maybe someone can validate this behaviour?

**Comment by nicm** [comment]
tmux has no control over the Option key, this is a Terminal.app problem. It
is presumably disabling it when an application takes over the mouse. Does
it work if you turn mouse off in tmux?

**Comment by olegTarassov** [comment]
Tmux works with option key in iTerm as expected and it works in terminal.app with mouse turned off. I can copy and highlight text using the option key.

**Comment by moritzdietz** [comment]
I agree with nicm that this is probably a terminal emulator problem.
See this thread: https://superuser.com/questions/660013/resizing-pane-is-not-working-for-tmux-on-mac
This could help you out.
